### ISSUE TITLE
Resource dependency doesn't work

### ISSUE THREAD

**Issue by dak1n1** [hashicorp/terraform-provider-kubernetes-alpha#52]
### Terraform Version and Provider Version

Terraform v0.12.26
kubernetes-provider-alpha 0.1.0

### Affected Resource(s)
kubernetes_manifest

### Terraform Configuration Files

Here is the config with an explicit dependency (`depends_on`).
https://gist.github.com/dak1n1/3dcb14290880d272918bee00a0066c2f

Here is the config with an implicit dependency (referencing the namespace name).
https://gist.github.com/dak1n1/5bf8e59302f44bb11f34c85ec2ff1d4f

### Debug Output

Debug trace for explicit dependency:
https://gist.github.com/dak1n1/70a8c5e28d0696f94e6fa063b96ee17c

Debug trace for implicit dependency:
https://gist.github.com/dak1n1/1f993d15185aa348ad96f24f9a793a8f

### Expected Behavior

It should create the namespace before it attempts to create the other resources.

### Actual Behavior

It errors saying the namespace does not exist.

```
Error: rpc error: code = Unknown desc = update dry-run for 'test/redis-master' failed: namespaces "test" not found
```

### Steps to Reproduce

1. Download one of the terraform configs attached to this issue and run `terraform apply`.

### Important Factoids

The resources work fine if I create them in a namespace that already exists (like `default`).

### References
- This might be fixed by `wait_for` https://github.com/hashicorp/terraform-provider-kubernetes-alpha/issues/12

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

**Comment by alexsomesan** [comment]
This is similar to https://github.com/hashicorp/terraform-provider-kubernetes-alpha/issues/57

The issue is a known limitation when using server-side planning (detailed in above issue). Will be resolved with the local planner (PR #41)

**Comment by dak1n1** [comment]
Closing this issue since we're tracking this in #57.
